packages/testing/src/BreakingChanges.spec.ts

```diff
@@ -117,6 +119,12 @@ describe('Breaking Changes', () => {
       )
 
       expect(encrypted).toMatchSnapshot()
+
+      const decrypted = await Message.decrypt(encrypted, decrypt, {
+        resolveKey: makeResolveKey(aliceDid),
+      })
+
+      expect(decrypted).toMatchSnapshot()
```

Comment:
wouldn't it be better to compare against `message` to make sure they are the same instead of making a second snapshot which you'd have to compare manually to the first?
```suggestion
      expect(decrypted).toMatchObject(message)
```

Reply:
Good idea! :+1: